src/components/Header/LargeHeader.js

```diff
@@ -0,0 +1,13 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import styles from './LargeHeader.styles'
+import { Color } from 'lib/theme'
+import { ReactComponent as Logo } from 'static/icons/logo.svg'
+
+export const LargeHeader = () => (
+  <div css={styles.root}>
+    <Logo fill={Color.WHITE} />
+  </div>
+)
+
+export default LargeHeader
```

Comment:
Double export 😄

Reply:
Just wanted to be extra sure it got exported 😆